feat(cdk): move `coercing` into lib (#730)
rx-angular/rx-angular: libs/cdk/coercing/README.md

# @rx-angular/cdk/coercing

[![npm](https://img.shields.io/npm/v/%40rx-angular%2Fcdk.svg)](https://www.npmjs.com/package/%40rx-angular%2Fcdk)
![rx-angular CI](https://github.com/rx-angular/rx-angular/workflows/rx-angular%20CI/badge.svg?branch=master)
[![Coverage Status](https://raw.githubusercontent.com/rx-angular/rx-angular/github-pages/docs/test-coverage/cdk/jest-coverage-badge.svg)](https://rx-angular.github.io/rx-angular/test-coverage/cdk/lcov-report/index.html)

> ## A small typed and documented convenience helper to apply performance improvements over coercing techniques.

Reviewer: ```suggestion
## Convenience helper to easily coerce reactive values
```

> `@rx-angular/cdk/coercing` is designed to help developers improve performance by leveraging reactive data streams. In particular it comes in handy when refactoring to a reactive setup,
> or if you want to have mixed code styles (imperative and reactive) appliead in an ergonimic way.
> Besides a well documented and typed API it also provides a set of marble diagrams for better visual understanding.

Reviewer: ```suggestion
`@rx-angular/cdk/coercing` is a small set of helpers designed to improve the DX when managing transformations between imperative and reactive values.
It provides a strictly typed API to conveniently coerce imperative setters into reactive streams.
The documentation also includes a set of marble diagrams for better visual understanding.
```

how is this really going to improve the performance?

## Key features

> <!-- - ✅ ... -->

Reviewer: ```suggestion
- turn imperative code into reactive streams
- strictly typed
```

## Demos:

> - [GitHub](https://github.com/BioPhoton/rx-angular-cdk-coercing)

Reviewer: ```suggestion
- [GitHub](https://github.com/rx-angular/rx-angular-cdk-coercing)
```

## Install

```bash

> npm install --save @rx-angular/cdk
> # or
> yarn add @rx-angular/cdk

Reviewer: ```suggestion
npm install --save @rx-angular/cdk/coercing
# or
yarn add @rx-angular/cdk/coercing
```

or am I wrong here?
